CMakeLists.txt

```diff
@@ -9,37 +9,15 @@ OPTION(ENABLE_WERROR "Build with -Werror" OFF)
 
 set (CMAKE_CXX_STANDARD 11)
 
-# Download and unpack googletest at configure time
-configure_file(CMakeLists.txt.in googletest-download/CMakeLists.txt)
-execute_process(COMMAND ${CMAKE_COMMAND} -G "${CMAKE_GENERATOR}" .
-  RESULT_VARIABLE result
-  WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/googletest-download )
-if(result)
-  message(FATAL_ERROR "CMake step for googletest failed: ${result}")
-endif()
-execute_process(COMMAND ${CMAKE_COMMAND} --build .
-  RESULT_VARIABLE result
-  WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/googletest-download )
-if(result)
-  message(FATAL_ERROR "Build step for googletest failed: ${result}")
-endif()
-
-# Prevent overriding the parent project's compiler/linker
-# settings on Windows
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  # GoogleTest 1.11.0
+  URL https://github.com/google/googletest/archive/e2239ee6043f73722e7aa812a459f54a28552929.zip
```

Comment:
You might want to see if the tag archive gives you the same code, it might be easier to decipher a tag format than a commit hash

https://github.com/google/googletest/archive/refs/tags/release-1.11.0.zip

Reply:
Good idea.  I was following the instructions [here](https://github.com/google/googletest/blob/main/docs/quickstart-cmake.md#set-up-a-project) which use a commit hash, but since I've tied the version to an actual release it makes sense to use the official tag as you suggest.  I'll give this a go.

Reply:
The two archives are indeed identical.  Thanks @1480c1.  This is much more readable.